### DISCUSSION TITLE
How to get WebSocketSession when using subcription?

### DISCUSSION THREAD

**Discussion by yapjiahong** [Netflix/dgs-framework#795] [Q&A]
Hi , I'm Raymond, I have a problem with subscription. 
Is it any where to get WebSocket Session ?
Because I would to know which subscription session is just established or closed.

**Comment by srinivasankavitha** [comment]
Could you describe what you would want to do with the session? The framework handles managing and tracking multiple sessions. So on the client side, you would get an indication of whether the session is established/closed in the response. Are you using the Apollo client or the Java client?
